lyon-amendment.md

```diff
@@ -0,0 +1,36 @@
+# The Lyon Amendment to the Lancaster Consensus
+
+🚧 This document is a *Work in Progress* and may change this week.
+
+## Minimum-support Perl
+
+The [Lancaster Consensus](lancaster-consensus.md) wrote:
+
+> Going forward, the Perl toolchain will target Perl 5.8.1, released September
+> 2003. This will allow toolchain modules to reliably use PerlIO and improved
+> Unicode support.
+>
+> Because of the many Unicode bug-fixes in early 5.8 releases, toolchain
+> maintainers reserve the right to later bump the minimum to 5.8.4 (which ships
+> with Solaris 10).
```

Comment:
The header already says it is a quote. I'd drop the >'s

Reply:
The blockquote is useful to see where the quote ends

Reply:
Kept it.